src/js/components/navigation.js

```diff
@@ -42,6 +44,23 @@ const navigation = behavior({
   [ CLICK ]: {
     [ OPENERS ]: toggleNav,
     [ CLOSERS ]: toggleNav,
+    [ NAV_LINKS ]: function() {
+      // A navigation link has been clicked! We want to collapse any
+      // hierarchical navigation UI it's a part of, so that the user
+      // can focus on whatever they've just selected.
+
+      // Some navigation links are inside accordions; when they're
+      // clicked, we want to collapse those accordions.
+      const acc = this.closest(accordion.ACCORDION);
+      if (acc) {
+        accordion.getButtons(acc).forEach(btn => accordion.hide(btn));
```

Comment:
Is it actually possible for a navigation to *not* contain an accordion?  I thought it was, but looking through the examples in our fractal site, I'm actually not sure anymore... If it's not possible, then I guess we might as well get rid of this conditional, maybe.